### PR TITLE
WAITP-1214: Mobile dashboard tabs fixes

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

Fixes issues 6-12 that were raised in this ticket https://linear.app/bes/issue/WAITP-1214/mobile-dashboard-and-map-tabs

---

### Screenshots:

image

### CODE REVIEW

File: packages/tupaia-web/src/AppProviders.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { theme } from './theme';
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
-      staleTime: 1000 * 60 * 5, // 5 minutes
+      staleTime: 1000 * 60 * 15, // 15 minutes
```
Comment (tcaiger) [inline packages/tupaia-web/src/AppProviders.tsx:15]: Made the stale time longer as the data can take a long time to load and is not updated often and this came up in testing.

File: packages/tupaia-web/src/features/Map/MapOverlaySelector/MapOverlayList.tsx [hunk 2]
```diff
@@ -167,7 +171,12 @@ export const MapOverlayList = () => {
     // when overlay changes, reset period to default
     urlSearchParams.set(URL_SEARCH_PARAMS.MAP_OVERLAY_PERIOD, DEFAULT_PERIOD_PARAM_STRING);
     setUrlParams(urlSearchParams);
+
+    if (toggleOverlayLibrary) {
+      toggleOverlayLibrary();
```
Comment (tcaiger) [inline packages/tupaia-web/src/features/Map/MapOverlaySelector/MapOverlayList.tsx:176]: close overlay after a change

File: packages/tupaia-web/src/features/Map/Map.tsx [hunk 3]
```diff
@@ -145,7 +150,7 @@ export const Map = () => {
           zoom={zoom as LeafletMapProps['zoom']}
           shouldSnapToPosition
         >
-          <TileLayer tileSetUrl={activeTileSet.url} showAttribution />
+          <TileLayer tileSetUrl={activeTileSet.url} showAttribution={false} />
```
Comment (acdunham) [inline packages/tupaia-web/src/features/Map/Map.tsx:153]: Just checking, was this something that just got changed to be true? If so, will setting it to false undo that fix?

File: packages/tupaia-web/src/features/Map/MapOverlaySelector/MobileMapOverlaySelector.tsx [hunk 4]
```diff
@@ -96,19 +103,27 @@ export const MobileMapOverlaySelector = ({
...
+  const { hasMapOverlays } = useMapOverlays(projectCode, entityCode);
+
   return (
     <Wrapper>
       {!overlayLibraryOpen && (
         <ButtonWrapper>
           <MapOverlayDatePicker />
-          <ExpandButton onClick={toggleOverlayLibrary} aria-controls="overlay-selector">
+          <ExpandButton
+            onClick={hasMapOverlays ? toggleOverlayLibrary : undefined}
```
Comment (acdunham) [inline packages/tupaia-web/src/features/Map/MapOverlaySelector/MobileMapOverlaySelector.tsx:115]: To be really pedantic, we could just disable the button when `hasMapOverlays` is false, which from an accessibility perspective is preferable I think

**Review by acdunham — APPROVED** [review]
LGTM, thanks for doing these changes. I have added a couple of questions, but otherwise pre-approving.

File: packages/tupaia-web/src/features/Map/Map.tsx (hunk 3 above)
Comment (tcaiger) [reply packages/tupaia-web/src/features/Map/Map.tsx:153]: It was but now I've upgraded the attribution to a custom one to match what was on tupaia before.

**Pull request merged**